MAUI/Themes/Keys.md

```diff
@@ -0,0 +1,109 @@
+---
+layout: post
+title: About Keys for Syncfusion Controls | Syncfusion
+description: This section contains the details about the keys which are mapped to the UI elements of all the controls
```

Comment:
description: This section contains the details about the keys mapped to the UI elements of all the controls.

Reply:
Changes addressed.